core/tests/test_utils.py

```diff
@@ -1355,6 +1355,34 @@ def assertRaisesRegex(  # pylint: disable=invalid-name
         return super(TestBase, self).assertRaisesRegex(
             expected_exception, expected_regex, *args, **kwargs)
 
+    # Here we used Mapping[str, Any] because, in oppia codebase TypedDict is
+    # used to define strict dictionaries and those strict dictionaries are not
+    # compatible with Dict[str, Any] type because of the invariant property of
+    # Dict type. Also, here value of `Mapping` is annotated as Any because this
+    # method can accept any kind of dictionaries for testing purposes. So, to
+    # make this method generalized for all test cases, we used Any here.
```

Comment:
```suggestion
    # Here we used Mapping[str, Any] because, in Oppia codebase TypedDict is
    # used to define strict dictionaries and those strict dictionaries are not
    # compatible with Dict[str, Any] type because of the invariant property of
    # Dict type. Also, here value of Mapping is annotated as Any because this
    # method can accept any kind of dictionaries for testing purposes. So, to
    # make this method generalized for all test cases, we used Any here.
```

Reply:
Done